domains/mrender.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "redirects to website",
+    "repo": "https://github.com/MrEnder0/MrEnder0.github.io",
+    "owner": {
+        "username": "MrEnder0",
+        "email": ""
+        "discord": "MrEnder#0001"
+    },
+    "record": {
+        "CNAME": "MrEnder0.github.io"
```

As said above, the website cannot be a "coming soon" page. If you want to can change it to 
```suggestion
        "URL": "https://github.com/MrEnder0"
```
Which will redirect to your GitHub profile.